client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,366 @@
+import os
+import time
+import stat
+import shutil
+
+import king_phisher.client.plugins as plugins
+
+from gi.repository import Gtk
+from gi.repository.GdkPixbuf import Pixbuf
+from gi.repository import Gdk
+from gi.repository import GObject
+
+import boltons.strutils
+import boltons.timeutils
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Josh Jacob']
+	title = 'SFTP Client GUI'
```

Comment:
I'd rename this to 'SFTP Client', the entire tool is GUI based so "GUI" is redundant.